clients/rust/src/lib.rs

```diff
@@ -11,10 +10,14 @@ extern crate bufstream;
 extern crate byteorder;
 extern crate bytes;
 mod sysadminctl;
-pub mod error_chain_generated_errors;
 #[cfg(test)]
 mod tests;
 
+#[macro_use]
+pub mod errors;
+use errors::SysadminErrorKind as ErrorKind;
+type Result<T> = errors::SysadminResult<T>;
```

Comment:
You shouldn't mix up module declarations, using statements, and type declarations like this. Things get messy pretty fast and it becomes difficult to figure out what was imported where, what types you are actually using, etc. INO, better to keep everything grouped so it's clear.